README.md

```diff
@@ -68,28 +68,50 @@ and integration tests including pact tests:
 After `npm install`, files will be created under both .git/hooks and .husky, that will automatically lint (and fix) any staged files in your commits, plus run a type check.
 
 ## Running the app
-The easiest way to run the app is to use docker compose to create the service and all dependencies. This starts the latest published docker container for hmpps-auth and redis, a fake manage-recalls-api (wiremock) and a local build of the manage-recalls-ui. 
+In easiest way to run ui and api is to use
 
-`docker compose up`
-
-OR use the following script to run in the background and ensure the fake-mange-recalls-api is up to date and all services start correctly:
+`./start-local-services.sh`
 
-`scripts/start-local.sh` 
+script in manage-recall-e2e project.
```

Comment:
It's manage-recalls-e2e-tests
maybe link to it?
https://github.com/ministryofjustice/manage-recalls-e2e-tests

Reply:
Fair, will add